specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/metricBatch.json

```
@@ -0,0 +1,449 @@
+          },
+          {
+            "$ref": "#/parameters/OrderByParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The **$filter** is used to reduce the set of metric data returned.<br>Example:<br>Metric contains metadata A, B and C.<br>- Return all time series of C where A = a1 and B = b1 or b2<br>**$filter=A eq ‘a1’ and B eq ‘b1’ or B eq ‘b2’ and C eq ‘*’**<br>- Invalid variant:<br>**$filter=A eq ‘a1’ and B eq ‘b1’ and C eq ‘*’ or B = ‘b2’**<br>This is invalid because the logical or operator cannot separate two different metadata names.<br>- Return all time series where A = a1, B = b1 and C = c1:<br>**$filter=A eq ‘a1’ and B eq ‘b1’ and C eq ‘c1’**<br>- Return all time series where A = a1<br>**$filter=A eq ‘a1’ and B eq ‘*’ and C eq ‘*’**."
+          },
+          {
+            "$ref": "#/parameters/ResourceIdList"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response to a metrics query.",
+            "schema": {
+              "$ref": "#/definitions/Response"
+            }
+          },
+          "400": {
+            "description": "The response to a metrics query that results in a bad request, with optional additional information.",
+            "schema": {
+              "$ref": "#/definitions/AdditionalInfoErrorResponse"
+            }
+          },
+          "default": {
+            "description": "Describes the format of Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Query Storage Metrics": {
+            "$ref": "./examples/MetricBatchExample.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ResourceIdList": {
+      "name": "resourceIds",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/ResourceIdList"
+      },
+      "description": "The comma separated list of resource IDs to query metrics for."
+    },
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The subscription identifier for the resources in this batch.",
+      "x-ms-parameter-location": "method"
+    },
+    "TimespanParameter": {
+      "name": "timespan",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "The timespan of the query. It is a string in the following format 'startDateTime_ISO/endDateTime_ISO'.",
+      "x-ms-parameter-location": "method"
+    },
+    "IntervalParameter": {
+      "name": "interval",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "format": "duration",
+      "description": "The interval (i.e. timegrain) of the query. The default value is PT1M.",
```

Recommend using the `default` attribute to define the default value rather than just stating it in the description.
```suggestion
      "default": "PT1M",
      "description": "The interval (i.e. timegrain) of the query.",
```